addons/misra.py

```diff
@@ -3851,6 +3851,8 @@ def misra_22_10(self, cfg):
         for token in cfg.tokenlist:
             if token.str == '(' and not simpleMatch(token.link, ') {'):
                 name, args = cppcheckdata.get_function_call_name_args(token.previous)
+                if not name:
```

Comment:
I have the feeling `if name is None:` would be more explicit. I would expect that the name is either None or a non-empty string.